ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -175,6 +176,97 @@ def _map_sample(self, sample, exclude_fields):
         return sample
 
 
+@dataclasses.dataclass
+class AudioExtensionTask:
+    audio_column_name: str = simple_parsing.field(default="audio", alias="-a")
+    asr_column_name: str = simple_parsing.field(default="sentence", alias="-A")
```

Comment:
maybe `text_column_name` to mirror `audio_column_name`? This confused me at first.